lint-staged.config.cjs

```diff
@@ -1,5 +1,4 @@
 module.exports = {
   '*.{js,jsx,ts,tsx,cjs,mjs}': ['eslint --fix', 'eslint'],
-  '**/*.ts?(x)': ['tsc-files'],
```

Comment:
+1 on this fix in case anyone else queries this, was the best solution we could find!